tests/system/model_monitoring/test_app.py

```diff
@@ -739,8 +739,21 @@ def _test_function_summaries(self) -> None:
                 )
             )
             evidently_func_summary = evidently_func_summary_list[0]
-            assert evidently_func_summary.stats["potential_detection"] == 1
-            assert evidently_func_summary.stats["detected"] == 0
+
+            evidently_stats = evidently_func_summary.stats
+
+            assert evidently_stats["potential_detection"] == 1
+            assert evidently_stats["detected"] == 0
+
+            # check the stream stats if stream is v3io
+            if isinstance(self.mm_stream_profile, DatastoreProfileV3io):
+                assert evidently_stats["stream_stats"]
+                assert evidently_stats["stream_stats"]["committed"] == 1
```

Comment:
do you have a test for when you don't aggregate across the shards/partitions?

Reply:
yes, the second part of the test which is based on the `function summary API` (and not `function summaries`) - https://github.com/mlrun/mlrun/pull/8310/files/184cdd6f9979ea70e89ea98d71ee3c14250418e3#diff-ffeb62cf7cad099646f03479931c0c5b709ca6697d6fd05836cb18e7d26a3569R794-R809